### PR TITLE
picks_bans & more cm features

### LINKED ISSUE #249: Show team/league on match pages
Think this requires a backend change to return this in the json

### LINKED ISSUE #493: Display picks and bans

### PR DESCRIPTION
**Pull request by blukai**
fixes #249 
fixes #493 

image

Radiant bans under their table, dire..

TODO in this PR:
- [x] Display teams;
- [x] Display league.

### CODE REVIEW

**Comment by blukai** [comment]
**< 716px**
image

**320px**
image

Random width: **484**
image

File: src/components/Match/Overview/Timeline.jsx [hunk 1]
```diff
@@ -164,7 +165,7 @@ export default ({ match }) => {
                         />
                       }
                       <ReactTooltip
-                        id={`event_${i}`}
+                        id={obj.type === 'firstblood' && !obj.key && !obj.player_slot ? 'notthistime' : `event_${i}`}
```
Comment (howardchung) [inline src/components/Match/Overview/Timeline.jsx]: what's this notthistime string for?

Comment (blukai) [inline src/components/Match/Overview/Timeline.jsx]: hehe, way to hide tooltip if first blood is not in objectives and and we can't get killer and target

File: src/components/Match/MatchHeader/MatchHeader.jsx [hunk 2]
```diff
@@ -25,16 +25,20 @@ export default ({ match, user, loading }) => {
...
           <Col xs={12} md={4} className={match.radiant_win ? styles.radiant : styles.dire}>
             <div className={styles.winner}>
-              {/* If need to display team info just make condition (logo, name (side))*/}
-              {match.radiant_win ?
-                <span>
+              {match.radiant_win
+                ? <span>
                   <IconRadiant />
-                  {strings.match_radiant_win}
+                  {match.radiant_team
```
Comment (blukai) [inline src/components/Match/MatchHeader/MatchHeader.jsx]: `Radiant Victory` in Russian will be `Победа сил света`, `Virtus.pro Victory` - `Virtus.Pro Победили`, same with Ukrainian, so `Victory` should have different ending and place 🦄

Comment (howardchung) [inline src/components/Match/MatchHeader/MatchHeader.jsx]: ah, ok.  I guess in the interest of localization we need separate strings then

File: src/components/Match/TeamTable.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import { IconRadiant, IconDire } from 'components/Icons';
+import Table from 'components/Table';
+import PicksBans from './Overview/PicksBans';
+import styles from './Match.css';
+
+const filterMatchPlayers = (players, team = '') => players.filter(player => (
+  (team === 'radiant' && isRadiant(player.player_slot)) || (team === 'dire' && !isRadiant(player.player_slot)) || team === ''),
+).sort((a, b) => a.player_slot - b.player_slot);
+
+export default ({
```
Comment (howardchung) [inline src/components/Match/TeamTable.jsx]: TeamTable now has a new function signature, it accepts the player array only (not match object)

File: src/components/Match/TeamTable.jsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      icon={<IconRadiant className={styles.iconRadiant} />}
+    />
+    <Table data={filterMatchPlayers(match.players, 'radiant')} columns={columns} />
+    <PicksBans data={match.picks_bans && match.picks_bans.filter(pb => pb.team === 0)} matchId={match.match_id} />
+    <Heading
+      title={match.dire_team ? match.dire_team.name : `${strings.general_dire} ${heading}`}
+      icon={<IconDire className={styles.iconDire} />}
+    />
+    <Table data={filterMatchPlayers(match.players, 'dire')} columns={columns} />
+    <PicksBans data={match.picks_bans && match.picks_bans.filter(pb => pb.team === 1)} matchId={match.match_id} />
```
Comment (howardchung) [inline src/components/Match/TeamTable.jsx]: Why are we rendering PicksBans in TeamTable?  Won't that make it show up under every set of tables?  We only want it on the overview right?

**Comment by dartharaknus** [comment]
@blukai Will you be opening this PR again?

**Comment by blukai** [comment]
Yes, I'll fix it

**Pull request merged**